README.md

```diff
@@ -37,12 +37,11 @@ make flash
 <img src="imgs/connection-nodemcu.jpg" alt="NodeMCU connection" width="365" height="270" />
 
 * The STM8 device of course needs G and 3.3V connections.
-* The current firmware doesn't yet support the RST line, so leave it unconnected.
-  (This is contrary to the indication in the picture.)
   <img src="imgs/connection-stm8.jpg" alt="STM8 connection" width="161" height="192" hspace="20" align="right" />
 * SWIM is connected to ESP8266 GPIO4 (e.g. D2 on NodeMCU) **via a 1kΩ pull-up
   resistor** (this is important since the builtin pullup resistor is not
   capable of pulling up the line fast enough!).
+* The RST line is connected to ESP8266 GPIO5 (D1 on NodeMCU).
```

Comment:
The SWIM part and NOTE below belong together. Either move the RST item above SWIM or below the NOTE.